<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buscar_win</class>
 <widget class="QDialog" name="Buscar_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="volver_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="opcion_p">
        <property name="text">
         <string>Patente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="opcion_n">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="input"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="info_b">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
